inst/odin/carehomes.R

```diff
@@ -783,11 +793,15 @@ update(D_tot) <- D_hosp_tot + D_comm_tot
 ##
 ## NOTE: the R_pre_tot sum sweeps out the second compartment used to
 ## model the mixture of exponentials.
-initial(R_pre_15_64) <- 0
-update(R_pre_15_64) <- sum(R_pre[4:13, ])
 
-initial(R_neg_15_64) <- 0
-update(R_neg_15_64) <- sum(R_neg[4:13])
+p_specificity <- user()
+N_tot_15_64 <- user()
+
+initial(prob_pos) <- 0
 
-initial(R_pos_15_64) <- 0
-update(R_pos_15_64) <- sum(R_pos[4:13])
+##prob_pos = prob_true_pos +
+##           prob_false_pos
+update(prob_pos) <- sum(new_R_pos[4:13]) / N_tot_15_64 +
```

Comment:
Can you move the huge comment above so that it is immediately above this line, so that the 4:13 does not come from nowhere?

Reply:
Done